Documentation/reference-guides/xfrm/index.rst

```diff
@@ -0,0 +1,301 @@
+.. only:: not (epub or latex or html)
+
+    WARNING: You are looking at unreleased Cilium documentation.
+    Please use the official rendered version released here:
+    https://docs.cilium.io
+
+.. _xfrm_guide:
+
+********************
+XFRM Reference Guide
+********************
+
+.. note:: This documentation section is targeted at developers and users who
+          want to understand the Linux XFRM subsystem. While reading this
+          reference guide may help broaden your understanding of Cilium, it is
+          not a requirement to use Cilium. Please refer to the
+          :ref:`getting_started` guide and :ref:`ebpf_datapath` for a higher
+          level introduction.
+
+Overview
+========
+
+IPsec encryption in the Linux kernel relies on `XFRM`_. XFRM is an IP framework intended for packet transformations, from encryption to compression. It is configured via a set of *policy* and *state* objects, which for IPsec, correspond to Security Policies and Security Associations.
+
+.. _XFRM: https://man7.org/linux/man-pages/man8/ip-xfrm.8.html
+
+XFRM Policies and States
+------------------------
+
+At a high-level, XFRM policies define what to accept and reject, whereas states define how to perform the encryption and decryption. Policies can match on the direction (``out``, ``in``, or ``fwd``), the source and destination IP addresses with CIDRs, and the packet mark. As an example, the following policy matches egressing packets with any source IP address, 10.56.1.X destination IP addresses, and ``0xcb93eXX`` packet marks. Policies default to allowing traffic as done here.
```

Comment:
The 'what' here in the first line is a bit vague. May want to say 'what network traffic' or something similar.